docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md

```diff
@@ -270,9 +270,29 @@ With Kubernetes, developers can progress from thinking about physical and virtua
 
 - Application health checks
 
+## Walkthrough 6: Deploy your Windows Containers-based apps to Azure App Service for Containers
```

Comment:
Add Scenario with diagram? Every other walkthrough has it. This is our most recommended path for Windows Containers and we should have that explicitly called out :)